go/base/context.go

```diff
@@ -225,7 +225,6 @@ func newMigrationContext() *MigrationContext {
 		InspectorConnectionConfig:           mysql.NewConnectionConfig(),
 		ApplierConnectionConfig:             mysql.NewConnectionConfig(),
 		MaxLagMillisecondsThrottleThreshold: 1500,
-		CutOverLockTimeoutSeconds:           3,
```

Comment:
Why is this removed? The default should remain at `3`

Reply:
Ah, I figured it would be best to have one place where the default value is defined - that would be in [main.go](https://github.com/github/gh-ost/blob/master/go/cmd/gh-ost/main.go#L89) with the rest of the flags.